src/lib/utils.ts

```diff
@@ -11,7 +11,7 @@ export function getLocation() {
 let ldClient;
 let ldClientReady = false;
 export function ldClientWrapper (key, user, options = {}) {
-  const queue = [];
+  const queue: Array<any> = [];
```

Comment:
Any reason this isn't `queue: any[]`

Reply:
Not really, will change it

Reply:
Normally you'd only use the Array<> syntax when a the generic type is more complex aka not a primitive. At least from memory that is what TSLint use to enforce